Lib/gftools/builder/recipeproviders/googlefonts.py

```diff
@@ -268,6 +282,18 @@ def build_STAT(self):
                 build_stat_step["needs"] = other_variables
             self.recipe[last_target].append(build_stat_step)
 
+    def build_Avar2(self):
```

Comment:
While I'm being pernickety, let's have standard snake case method names (`build_avar2`). (`build_STAT` is just about OK because that's the table name.)